Scripts/GameManagement/LobbyManager.gd

```diff
@@ -1,6 +1,6 @@
 extends Node
 
-var Player = preload("res://Entities/Character/Player/Player.tscn")
+var Player = preload("res://Entities/Character/Player/Classes/Soldier.tscn")
```

Comment:
Do we need preloading Soldier in lobby?

Reply:
No, we don't. I forgot that I removed a feature that required it.